migrations/20220428092250_initial_tables.js

```diff
@@ -18,6 +18,22 @@ exports.up = async (knex) => {
     def.primary(['id'])
   })
 
+  await knex.schema.createTable('orders', (def) => {
```

Comment:
Do you mean each table has to be built individually? Sometimes you need to access more than one table so personally I'm happy with how it is at the moment, not sure what others think.

Reply:
so if you run `npx knex migrate make foobar` it will create a dated migration file `foobar_{timestamp}`. @n3op2 is right this is how we should be doing this

Reply:
Done